docs/send-data/installed-collectors/sources/script-source/cron-examples-reference.md

```diff
@@ -39,7 +39,7 @@ There are several special characters that are used to specify values:
 |:--|:--|:--|
 | \* | All values. | `*` in the minute field means every minute. |
 | ? | No specific value in the day of month and day of week fields. | `?` specifies a value in one field, but not the other. |
-| \* | A range. | `10-12` in the hour field means the script will run at 10, 11 and 12 (noon). |
+| - | A range. | `10-12` in the hour field means the script will run at 10, 11 and 12 (noon). |
```

Comment:
```suggestion
| - | A range. | `10-12` in the hour field means the script will run at 10, 11, and 12 (noon). |
```

Reply:
@JV0812, I'll go ahead and commit your suggestion.